Fabric.Terminology.API/Extensions.ValueSetService.cs

```diff
@@ -11,7 +11,7 @@ public static partial class Extensions
     {
         public static Attempt<IValueSet> Create(this IValueSetService service, ValueSetCreationApiModel model)
         {
-            return service.Create(model.Name, model, model.CodeSetCodes.Select(code => code.ToCodeSetCode()));
+            return service.Create(model.Name, model, model.CodeSetCodes.Select(code => code.ToCodeSetCode()).ToList());
```

Comment:
We need to ensure that the CodeSetCodes is not an empty collection.  This is related to issue #39 (which deals specifically with the counts) but it looks like Clint may have added some additional constraints.  

Maybe something like:

      return model.CodeSetCodes.Any() ? 
           Attempt<IValueSet>.Successful(service.Create(model.Name, model, model.CodeSetCodes.Select(code => code.ToCodeSetCode())) :
          Attempt.<IValueSet>.Failed(new ValueSetOperationException("Attempted to create a value set with an empty list of codes");

Reply:
Should have followed this through further ... it's handled in the service!